src/main/java/org/akaza/openclinica/controller/dto/SdvItemDTO.java

```diff
@@ -0,0 +1,51 @@
+package org.akaza.openclinica.controller.dto;
+
+import java.util.Date;
+
+public class SdvItemDTO {
+    private String name;
+    private String briefDesc;
```

Comment:
it's always better to have full name. rename this to briefDescription

Reply:
Updated as per the Comments